src/user/formats/hal.ts

```diff
@@ -83,7 +83,10 @@ export function item(user: User, privileges: PrivilegeMap, hasControl: boolean,
         href: group.href,
         title: group.nickname,
       })),
-
+      'userinfo': {
+        href: `${user.href}/userinfo`,
+        title: 'User Info'
+      },
```

Comment:
It's not needed to make a new endpoint / route for userinfo, because we're just going to return the data as part of the User resource.